src/metabase/api/activity.clj

```diff
@@ -55,18 +54,10 @@
        (cond-> {}
          (seq dataset-ids) (assoc "dataset" (set dataset-ids))
          (seq card-ids')   (assoc "card" (set card-ids')))))
-   (when-let [dashboard-ids (get referenced-objects "dashboard")]
-     (let [id->page?                            (db/select-id->field :is_app_page Dashboard
-                                                                     :id [:in dashboard-ids])
-           {page-ids true dashboard-ids' false} (group-by (comp boolean id->page?)
-                                                          ;; only existing ids go back
-                                                          (keys id->page?))]
-       (cond-> {}
-         (seq page-ids)       (assoc "page" (set page-ids))
-         (seq dashboard-ids') (assoc "dashboard" (set dashboard-ids')))))
-   (into {} (for [[model ids] (dissoc referenced-objects "card" "dashboard")
```

Comment:
There are no pages any more, so dashboards are handled like the other simple types at line 60.